host/windows/hostthread.c

```diff
@@ -13,25 +13,26 @@
 **==============================================================================
 */
 
-int oe_thread_create(oe_thread* thread, void* (*func)(void*), void* arg)
+oe_result_t oe_thread_create(oe_thread* thread, void* (*func)(void*), void* arg)
 {
-    *thread = CreateThread(NULL, 0, func, arg, 0, NULL);
-    return *thread == NULL;
+    DWORD (*start_routine)(void*) = (DWORD(*)(void*))func;
```

Comment:
Consider a typedef for the `start_routine` function pointer type to make this more readable.